dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/MvcTestConfig.java

```diff
@@ -232,6 +234,12 @@ public void addInterceptors( InterceptorRegistry registry )
         registry.addInterceptor( new RequestInfoInterceptor( requestInfoService ) );
     }
 
+    @Override
+    public void addFormatters( FormatterRegistry registry )
+    {
+        registry.addConverter( new StringToOrderCriteriaListConverter() );
+    }
```

Comment:
do we need this right now? I don't see any web MVC test. Maybe we can add it when it is required. Maybe i am wrong